Bloggist/src/objects/UserProvisioningRequest.object

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
```

Comment:
Please remove the objects that aren't part of your project.

Reply:
This is set like this probably because clicking in the custom objects tab when updating metadata from MavensMate. It will be fixed